recipe.py

```diff
@@ -42,12 +42,18 @@ def get_ingredients(self):
     def set_ingredients(self,ingredients):
         self.ingredients.append(ingredients)
 
+    def remove_ingredient(self,ingredient):
+        self.ingredients.remove(ingredient)
+
+
     def get_instructions(self):
         return self.instructions
 
     def set_instructions(self, instructions):
         self.instructions.append(instructions)
```

Comment:
Note you can append the same instruction multiple times, but only remove all of them at once because they are duplicates.

Reply:
I will look at how to leave 1 of the instructions if removing duplicates